### DISCUSSION TITLE
[邮件订阅] 收集邮件

### DISCUSSION THREAD

**Discussion by tangly1024** [tangly1024/NotionNext#2077] [Ideas]
在此讨论网站接入邮件订阅功能的想法

**Comment by tangly1024** [comment]
目前已知的方案；


1. 用mail chimps的表单进行邮件营销 https://mailchimp.com/
2. 用notion form 进行表单收集，群发邮件需要自助 https://notionforms.io/login

**Comment by tangly1024** [comment]
网友提出使用 类似 https://pt.plus/ 这种完整的付费订阅逻辑。
对方采用的是ghost建站方案。

**Comment by dreamer2q** [comment]
订阅邮箱后，那些操作、事件会触发邮件发送呢？
邮件发送支持自定义触发吗？

**Comment by alanjustalan** [comment]
> 目前已知的方案；
> 
> 1. 用mail chimps的表单进行邮件营销 https://mailchimp.com/
> 2. 用notion form 进行表单收集，群发邮件需要自助 https://notionforms.io/login
mail chimps有教程吗？我卡在最后一步不懂怎么弄了

**Comment by tangly1024** [comment]
https://github.com/tangly1024/NotionNext/issues/1236

**Comment by raygorous** [comment]
这个issue目前有什么进展吗？

**Comment by Willywang8216** [comment]
問問目前的計畫，NOTION 現在有自己的FORM 了，不過沒有recaptcha 用起來應該會很慘，可以加入比如typeform 之類的嗎??? 相較mailchimp 有brevo 等更好用的服務 @tangly1024
